eng/pipelines/templates/jobs/live.tests.yml

```diff
@@ -78,35 +78,35 @@ jobs:
       - template: ../steps/common.yml
 
       - ${{ if ne(parameters.ServiceDirectory, '') }}:
-        - template: /eng/common/TestResources/build-test-resource-config.yml
-          parameters:
-            SubscriptionConfiguration: ${{ parameters.CloudConfig.SubscriptionConfiguration }}
-            SubscriptionConfigurations: ${{ parameters.CloudConfig.SubscriptionConfigurations }}
-            EnvVars: ${{ parameters.EnvVars }}
-
-        - ${{ if parameters.TestResourceDirectories }}:
-          - ${{ each directory in parameters.TestResourceDirectories }}:
-            - template: /eng/common/TestResources/deploy-test-resources.yml
-              parameters:
-                ${{ if or(parameters.Location, parameters.CloudConfig.Location) }}:
-                  Location: ${{ coalesce(parameters.Location, parameters.CloudConfig.Location) }}
-                ServiceDirectory: '${{ directory }}'
-                SubscriptionConfiguration: $(SubscriptionConfiguration)
-                ArmTemplateParameters: $(ArmTemplateParameters)
-                UseFederatedAuth: ${{ parameters.UseFederatedAuth }}
-                ServiceConnection: ${{ parameters.CloudConfig.ServiceConnection }}
-                SubscriptionConfigurationFilePath: ${{ parameters.CloudConfig.SubscriptionConfigurationFilePath }}
-        - ${{ if not(parameters.TestResourceDirectories) }}:
-          - template: /eng/common/TestResources/deploy-test-resources.yml
+          - template: /eng/common/TestResources/build-test-resource-config.yml
             parameters:
-              ${{ if or(parameters.Location, parameters.CloudConfig.Location) }}:
-                Location: ${{ coalesce(parameters.Location, parameters.CloudConfig.Location) }}
-              ServiceDirectory: '${{ parameters.ServiceDirectory }}'
-              SubscriptionConfiguration: $(SubscriptionConfiguration)
-              ArmTemplateParameters: $(ArmTemplateParameters)
-              UseFederatedAuth: ${{ parameters.UseFederatedAuth }}
-              ServiceConnection: ${{ parameters.CloudConfig.ServiceConnection }}
-              SubscriptionConfigurationFilePath: ${{ parameters.CloudConfig.SubscriptionConfigurationFilePath }}
+              SubscriptionConfiguration: ${{ parameters.CloudConfig.SubscriptionConfiguration }}
+              SubscriptionConfigurations: ${{ parameters.CloudConfig.SubscriptionConfigurations }}
+              EnvVars: ${{ parameters.EnvVars }}
+
+          - ${{ if parameters.TestResourceDirectories }}:
+              - ${{ each directory in parameters.TestResourceDirectories }}:
+                  - template: /eng/common/TestResources/deploy-test-resources.yml
+                    parameters:
+                      ? ${{ if or(parameters.Location, parameters.CloudConfig.Location) }}
```

Comment:
What is going on here lol. Is this an intentional ternary or some more weird auto-formatting?

Reply:
Thanks for spotting this. We should probably exclude `eng/` from auto formatting.